packages/presentation/lib/screen/home/home_bloc.dart

```diff
@@ -0,0 +1,117 @@
+import 'dart:math';
+
+import 'package:domain/models/check_number_params.dart';
+import 'package:domain/usecase/check_number_usecase.dart';
+import 'package:domain/usecase/generate_number_usecase.dart';
```

Comment:
usecase - use_case -> editor swears at the name

Reply:
I ignore this word and add it to dictionary. This solve the problem.